src/components/Login.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Navbar from "./Navbar";
```

Comment:
remove navbar  from login and signup page

Reply:
Okay